strings/tcg/credits.md

```diff
@@ -0,0 +1,56 @@
+Major thanks to all contributors to the TCG project, in particular ArcaneMusic for his code contributions, and LemonintheDark for his backend system, without which none of this would be possible.
+Additional thanks to Shaps, for lore advice.
+Description Credits
+2020 CORE SET
+	EOBGames:
```

Comment:
Is this a thing we do? I thought we took care of credits in the changelog?

Reply:
I did it specifically for this PR since a good number of people contributed card descriptions, and I wanted more granular credit for that. I can remove it if need be.

Reply:
I'm only seeing 3 names here and one of them is yours.

Reply:
It looks like there are 3 names here and one of them is yours.

Honestly it kind of feels unnecessary. What about this code justifies having a credits.md and names in it more than any other contribution that has come before it? I've seen names be rejected from code comments, it's a pretty firm policy from what I understand.